### DISCUSSION TITLE
the problem about grid

### DISCUSSION THREAD

**Discussion by husq-diandian** [antvis/X6#2235] [Q&A]
Can the mesh grid be rotated 45 degrees

**Comment by NewByVector** [comment]
You can customize the grid. refer [this](https://x6.antv.vision/zh/docs/api/registry/grid#registry).

**Reply by husq-diandian** [reply]
customize the grid cannot set angle.

**Reply by husq-diandian** [reply]
![grid](https://user-images.githubusercontent.com/52273012/174511669-23e745b7-7149-4cf4-88ba-1dae8ca3c51f.png)

**Reply by NewByVector** [reply]
The graphics for the rotation effect need to be implemented using SVG graphics.
